app/v2/errors.py

```diff
@@ -85,14 +85,15 @@ def to_dict_v2(self):
 def register_errors(blueprint):
     @blueprint.errorhandler(InvalidRecipientError)
```

Comment:
this also catches `InvalidEmailError` and `InvalidAddressError` - they won't have the `get_legacy_v2_api_error_message` function on them 🤔 

does this work correctly for invalid emails? or does this now crash

i think we'll want a separate error handler for `InvalidPhoneError` (and then decorate this function twice - once for InvalidLetter and one for InvalidEmail?)

Reply:
No tests have failed ... (assuming we have tests for invalid emails)

Reply:
```python
    @blueprint.errorhandler(InvalidPhoneError)
    def invalid_phone_format(error):
        return .... (get_legacy_v2_api_error_message)

    @blueprint.errorhandler(InvalidRecipientError)
    def invalid_format(error):
        return .... (str(error))
    ```

Reply:
Added the InvalidPhoneError handling.

Do we want to think about tests for emails now or later?

Reply:
lets worry later